system-parachains/asset-hubs/asset-hub-kusama/src/lib.rs

```diff
@@ -950,7 +951,7 @@ parameter_types! {
 /// We allow root and the `StakingAdmin` to execute privileged collator selection operations.
 pub type CollatorSelectionUpdateOrigin = EitherOfDiverse<
 	EnsureRoot<AccountId>,
-	EnsureXcm<IsVoiceOfBody<GovernanceLocation, StakingAdminBodyId>>,
+	EnsureXcm<IsVoiceOfBody<RelayChainLocation, StakingAdminBodyId>>,
```

Comment:
should not we have here the AH location or we are sure its gonna be converted to `Root` and pass with `EnsureRoot<AccountId>`?

Reply:
> should not we have here the AH location or we are sure its gonna be converted to `Root` and pass with `EnsureRoot<AccountId>`?

Right, thank you, AH allows StakingAdmin from RC, but we also need to allow `StakingAdmin` from the local OpenGov.
Fixed by: https://github.com/polkadot-fellows/runtimes/pull/877/commits/c5e833bb406214701cc711f429e07f655e7cb2ed